src/pages/storage/CustomVolumeModal.tsx

```diff
@@ -21,6 +24,8 @@ const CustomVolumeModal: FC<Props> = ({
   project,
   onFinish,
   onCancel,
```

Comment:
I wonder if we should rename those to `onGoBack` or something that is avoiding the ambiguity between onCancel and onClose.

Reply:
I think the logic originates from the parent, the modal does not always "go back" so `onCancel` seems like a better fit. For example, when the modal is opened by clicking on the edit icon for an existing volume device, clicking the cancel button will close the modal